.travis.yml

```diff
@@ -3,7 +3,8 @@ node_js:
   - '8'
 env:
   global:
-    - secure: "iEU6P9cbuDOcZ2klMIgkIucGBTxUHp7A7ZcQTHBdmmwQlTCw2nqF1Mh9145Uycu3os64my4QCokHlPlgLgxvMDou+kLDJqx0EwSE7brZs8+NkefeEBTolOOCpnhfcMqhtO9eDznTUEZZKXylRIo/ZRqVcXSs6VRexG6nY+LjVX/ss0ISEs+ih8atauK8m61zJxlrQjHGP0jPxXWAK1hAXQW+qU0/KDN+UPYZmwoV4PSDB43LAN0gciFnUIcZZXSYqSLma5u7oX8MEH1cGC7IrvGxvEumYUCvREc+z49N1qoFM3TP4Mb92aHOcKX3VwUubRUX3PS2WxnO9hNmUQAif61+NTdf3et3jJ8lPmU351X9y51NYi+INXgljTLjjm+BvR2LuDT2iuWHzTFrYxzc9bD4uR/dukzDD0SYAvmYVzqrGqhhXpMvKB3Bb/r5lNKYVwm+OsqqQNxG2XGrbon8YnY2l4cbTyd5eDgHVqIrZKL5zb59LssF4rBL6IlMl4kx88dfn7a6NSGRQYxqap0OwmwoWCDGfBiEEjcwh3lwRlP2Hu6r4b0nN2n8N1wH10JQQlZnMhqQFB301f8hQgm5X0+aQZ6HrIgBcZC9p6TVNCMHpfCjdgma1k4YfMRFzCM5qw1gZVamFCvX6w1Q5MIJguxJ+GKjZvkp5dZWdWdysOI="
+    # kosssi GITHUB_TOKEN
+    - secure: "c4Mdc0HhRtDzi4U3ykwPawh7NvfkpDRzhQlBn0tG31YjZQ1Q2/IUHfKB/nQ1e/xMin8lg4UT/S3UxQZv3+9dn87LfrEBDw7HAmTOQi8uay2Nvvgjo3lpj3DA/YDbTXtvrVx529CfRohCYsVSAJy8QVyFnN1XXyHd2jV3P1JtWNastpKVqqVPXvO8O53GXQvLO4W2z738Z4JjzfCxeItGxhjc45kxwHvN//+VXO9qvJ31IsUPjGopReCf6SaATIVa5v3s6aNvbw83t5eB87O28CIhmjB2naG990D8mEZDsdHXCiyTS8l/qyEtZL8lxnWpySL+m0p0LZ31CFynq25fB1ZC1KdQQoxHZxpxriUxR1MTcU8QrCl0dAmm+aEGRh2b/qp0vfKnB21YayxWJkquli4PeD/MBQFTd6CHH8ZWMOvlHF4UR29Hh5A9l1jXbJfSwulbIF8Pv8ThqY/YNjD0qCQ9D19/lsXhghMBGUEsZKgUvdpgZ2YjE6eBwRFFe0bp85UBa1WaX3Uam5cMaKfCB75Vo3cP/m6VyB6QXSzmdb36KmWF8zY4TBIEVVsis4lFXV7ZDQ9Wpr2bUzq13q3YNngzF2/8+dPs4g+aqIRhz9LniHHDLKzd3gM9adXd5Befmj3GFcVvYGI/6JJuwc6nmxMzKHSv/348knebx44O4AQ="
```

Comment:
Why doe it need to be changed ?

Reply:
J'ai fait un update d'un token github qui impactait seulement ce konnector ;)